### PR TITLE
Update docker-compose

### PR DESCRIPTION
**Pull request by iyoungman**
* docker-compose.yml 수정 test

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -17,3 +17,4 @@ services:
     ports:
       - 27017:27017
 
+
```
Comment (iyoungman2) [inline docker-compose.yml:20]: ok

**Review by iyoungman2 — APPROVED** [review]
review ok

**Review by iyoungman2 — APPROVED** [review]

File: docker-compose.yml (hunk 1 above)
Comment (iyoungman2) [reply docker-compose.yml:20]: ok2

**Comment by iyoungman** [comment]
thank you

**Pull request closed without merging**